### ISSUE TITLE
Need to handle multi-variable datasets

### ISSUE THREAD

**Issue by senesis** [senesis/climaf#9]
It would be useful to let dataset be multi-variable, in order to cope with datafile organizations where variables are grouped in files (such as for Nemo model diags). This would save file operations when such  groups of variable are used together by some operators. This would however break the regularity of CliMAF dataset model

**Comment by oliviermarti** [comment]
We in someway emulates that with Jerome by using dictionnaries :

```
O1T03V05={}
for Var in ['tos', 'sos', 'to', 'thetao', 'so', 'mlotst', 'sic' ] :
    O1T03V05[Var] = ds(project="IGCM_OUT", simulation="O1T03V05", period="1850-1859", experiment="piControl", login="p86mart", frequency="seasonal", model="IPSLCM6", variable=Var )

print O1T03V05['to'].baseFiles()
```
Seems convenient...

Olivier

**Comment by senesis** [comment]
Yes, fine, thanks. But this does not adress the objective to 'save file operations when such groups of variable are used together by some operators' and are also goruped in files (e.g. : Cdftool operator 'cdftransport' can make use of a single file where averages of U.S, V.S, U.T and V.T are grouped)

**Comment by senesis** [comment]
A cost-effective solution was implemented in version V0.10, see the news and example cdftransport.py
